### ISSUE TITLE
GoldenEye: Source Crashing on Start

### ISSUE THREAD

**Issue by LukeRhok** [GameServerManagers/LinuxGSM#2053]
**-Issue:** 
For some reason this game always crashes on it's startup. Looking around the configs, I can't find anything I did wrong in the configurations. A friend of mine tried the same server in a virtual machine, using the same distro, and he gets the same results. Is this a bug on the server or is there something that I might be missing or done know about? Let me know if any additional information is needed.
**-Game Server:** 
GoldenEye: Source
**-Distro:** 
Linux Mint 19 MATE (equivilient to Ubuntu 18.04)
**-Console Log:** 
https://drive.google.com/file/d/1YeyLbof89GOwcDLDHqCcRu6Q8fnwMDK_/view?usp=sharing
**-GameServer details:**
 https://hastebin.com/eqahuzaley

**-Notes:** 
I posed this issue on the Discord server and dgibbs said that he was going to dig into this issue and test it out.  If any addional information is needed please let me know.

**Comment by dgibbs64** [comment]
Yup its a specific issue that the goldeneye source team will have to fix. It will need to be reported to them

**Comment by LukeRhok** [comment]
> Yup its a specific issue that the goldeneye source team will have to fix. It will need to be reported to them

Sounds good, I appreciate you taking a look. I'll report the issue to them to see if they can help me solve this problem.

Thanks again for your help.

**Comment by dgibbs64** [comment]
Cheers @LukeRhok please inform us if this gets fixed.

**Comment by dgibbs64** [comment]
Website needs updating to r
Advised users 18.04 does not work

**Comment by LukeRhok** [comment]
I just want to note too that the game also seemed to crash on startup after installing it in Ubuntu 16.04 which is running in a virtual box VM.  My thought as to the reason why the game no longer works on Ubuntu 16.04 for me could possibly be due to the 5.0.6 patch that was released in August 2017. I'm not sure if anyone else's results are different currently?
In addition to that, I noticed that on the GoldenEye: Source website, under their troubleshooting section, they ask you to try installing the 32 bit 2012 Visual C++ Redistributable Package if "the game crashes on menu load".  I will reach out to them before I attempt to install that on Linux.

**Comment by LukeRhok** [comment]
I apologize for not leaving any updates for this in a while, I've been out of my state for a few days and have been very busy with more important life matters that came up. 
However, I was able to get ahold of someone on the GolenEye: Source team and they said "We've stopped supporting the Linux build natively and wrote a guide on how to run the Windows server under Wine. http://wiki.geshl2.com/goldeneye/server_install_linux ".  Unfortunately not the most helpful. I did attempt to follow their guide and install the GoldenEye: Source on a Linux machine, but as of right now I haven't been able to confirm whether it actually fully works or not because I'm unable to see any console output due to the way they have you install the game in Linux. However, I'm guessing that it's not working correctly given that I'm not able to connect to the server locally, but I'm unable to confirm anything until I can get some console data or any sort of helpful log. 
That is where I'm currently at right now, just haven't had the time to sit down and look at this some since.
